prog/vm/github_runner.rb

```diff
@@ -247,6 +247,18 @@ def setup_info
     # Remove comments and empty lines before sending them to the machine
     vm.sshable.cmd(command.gsub(/^(\s*# .*)?\n/, ""))
 
+    if github_runner.installation.project.get_ff_all_servers_dnsmasq
+      # replace server=9.9.9.9@IP_ADDRESS with server=2606:4700:4700::1111
+      # and server=149.112.112.112@IP_ADDRESS with server=2606:4700:4700::1001
+      command = <<~COMMAND
+        sudo sed -i 's/^server=9.9.9.9@.*/server=2606:4700:4700::1111/' /vm/#{vm.inhost_name}/dnsmasq.conf
+        sudo sed -i 's/^server=149.112.112.112@.*/server=2606:4700:4700::1001/' /vm/#{vm.inhost_name}/dnsmasq.conf
+        echo "all-servers" | sudo tee -a /vm/#{vm.inhost_name}/dnsmasq.conf
+        sudo systemctl restart #{vm.inhost_name}-dnsmasq
+      COMMAND
+      vm.vm_host.sshable.cmd(command)
```

Comment:
I'd try to avoid this additional `sshable.cmd(command)`. Similar to `github_runner.installation.cache_enabled`, why not just add these commands to `command` and then execute it all here in one go https://github.com/ubicloud/ubicloud/blob/f56ed7c3787d1e76c00c459a5c0c17a219c984bb/prog/vm/github_runner.rb#L248

Reply:
but this one is run against vm_host

Reply:
ah, I see, of course. then we can't combine the ssh calls.